app/views/jobseekers/job_applications/review.html.slim

```diff
@@ -39,3 +39,11 @@
       - if vacancy.listed?
         = f.govuk_submit t("buttons.submit_application") do
           = f.govuk_submit t("buttons.save_and_come_back"), secondary: true
+
+  - if current_jobseeker.job_applications.not_draft.none?
+    .govuk-grid-column-one-third
+      = render StepsComponent.new title: "Application Steps", classes: "govuk-!-margin-top-6" do |component|
```

Comment:
Why can't we reuse the steps partial?

Reply:
The review page isn't a part of the wizard!